contracts/btc-staking/src/validation/mod.rs

```diff
@@ -49,6 +50,42 @@ fn verify_pop(
     Ok(())
 }
 
+#[cfg(feature = "full-validation")]
+fn get_pks(
```

Comment:
Nit: this helper should perhaps be better named `deserialize_pks`, `decode_pks` or so.